docs/OV_Runtime_UG/multi_device.md

```diff
@@ -19,7 +19,7 @@ The setup of Multi-Device can be described in three major steps:
 
```

Comment:
could you please update the python version as well?

Reply:
updated

Reply:
updated